LocalSettings.php

```diff
@@ -1552,6 +1552,7 @@
 		'+showroomwiki' => array( 'png', 'gif', 'jpg', 'jpeg', 'doc', 'xls', 'pdf', 'ppt', 'tiff', 'bmp', 'docx', 'xlsx', 'pptx'),
 		'+techeducationwiki' => array( 'docx', 'doc', 'odt', 'ods', 'odp', 'ppt', 'xls', 'xlsx','xml'),
 		'+themirrorwiki' => array( 'mp3'),
+		'+thenetwiki' => array( 'zip', 'rar', 'dae'),
```

Comment:
You also have to add the ones from the default too.

Reply:
@paladox, no the + means it’s appending the default